0122-amend-service-provider-hex-boosting.md

```diff
@@ -47,12 +47,33 @@ This HIP aims to do three (3) things to amend HIP-84 Service Provider Hex Boosti
    - Boosted rewards will no longer benefit from the overflow from the Data Transfer Bucket, as they will be solely dependent on their own bucket.
    - Because of this 10% allocated Hex Boosting Bucket, this would enable a dynamic maximum / ceiling for hex boosts depending on the number of hexes that are actively pulling for boosted rewards. As of writing this HIP, the allocation of PoC to boosted hexes is far above 10%. This will cause boosted hexes to natively get less rewards until the bucket reaches the 10% equilibrium.
    - The Helium Planner has always stated "Up to ##x"; so this precedent has already been set that none of the boosted hexes multipliers have been guaranteed that exact multiplier, and it could be lower.
-
-2. Reduce the minimum time for Service Provider Hex Boosting to 3 months.
+   - Mobile rewards are calculated based on the total CPs for that epoch and the PoC bucket of rewards (sans the Mobile due to unused Data rewards). The total Mobile rewards due to the CPs from the boosted hexes are then checked against the 10% Mobile emissions for that epoch. If the number of mobile tokens is higher that the 10% amount, the Mobile rewards for the boosted CPs are reduced  to meet the 10% cap. Mobile rewards from base CPs are then re-calculated after the unused Data rewards are added back into the PoC reward bucket.
+      - Example:
+
+         Total Poc bucket: 1,000,000 MOBILE  (500,000 boosted PoC and 500,000 normal PoC)
+   
+         Total Coverage Points: 100,000
+      
+         Total Boosted Coverage Points: 75,000
+      
+         Total Normal Coverage Points: 25,000
+      
+         1,000,000 / 100,000 = 10 MOBILE per CP, 10 * 75,000 exceeds the boosted PoC cap so we have to calculate the MOBILE per CP differently for boosted vs non boosted
+     
+         Boosted: 500,000 / 75,000 = 6.666666667 MOBILE per boosted CP
+     
+         Normal: 500,000 / 25,000 = 20 MOBILE per normal CP
+     
+         For a hotspot with 500 CP, 100 normal and 400 boosted
+     
+         (100 x 20) + (400 * 6.666666667) = 4666.666667
```

Comment:
```suggestion
           - Assumptions
             - Total Poc bucket: `1,000,000 MOBILE` (500,000 boosted PoC and 500,000 normal PoC)
             - Total Coverage Points: `100,000`
             - Total Boosted Coverage Points: `75,000`
             - Total Normal Coverage Points: `25,000`
           - Calculation
             - `1,000,000 / 100,000 = 10 MOBILE per CP, 10 * 75,000` exceeds the boosted PoC cap so we have to calculate the MOBILE per CP differently for boosted vs non boosted
             - Boosted
               ```
               500,000 / 75,000 = 6.666666667 MOBILE per boosted CP
               ```
             - Normal
               ```
               500,000 / 25,000 = 20 MOBILE per normal CP
               ```
             - For a hotspot with 500 CP, 100 normal and 400 boosted
               ```
               (100 x 20) + (400 * 6.666666667) = 4666.666667
               ```
```

Reply:
i think this would format better.